docs/book/v2/quick-start.md

```diff
@@ -0,0 +1,149 @@
+# Quick Start
+
+## Mapping Routes to Middleware
+
+The first step is to map a route to PSR-7 middleware. This looks like any other
+[routing](https://docs.laminas.dev/laminas-mvc/routing/) configuration,
+with small changes: the `controller` key in the routing options has to be the `\Laminas\Mvc\Middleware\PipeSpec` class
```

Comment:
The same here.

Reply:
okay, I changed it everywhere I found a FQCN that started with a backslash.
But isn't it problematic (espacially in this case) that readers copy/paste the class into their config which probably _is_ namespaced and run into an error?

Reply:
> But isn't it problematic (espacially in this case) that readers copy/paste the class into their config which probably _is_ namespaced and run into an error?

The target group of the documentation are experienced PHP developers and each IDE also displays the error.

Reply:
> The target group of the documentation are experienced PHP developers and each IDE also displays the error.

Yes, then it's not a problem 🙂